examples/nodejs/testingbot/bot.js

```diff
@@ -0,0 +1,179 @@
+"use strict";
+const {
+    MediaStream,
+    nonstandard: { RTCAudioSink }
+} = require("wrtc");
+const {
+    HiFiAudioAPIData,
+    Point3D,
+    OrientationEuler3D,
+    HiFiCommunicator,
+    HiFiConstants
+} = require("hifi-spatial-audio");

```

Comment:
Switch to ESM import syntax, or else this example will not work with the latest version of our client library :)

```suggestion
import {
    HiFiAudioAPIData,
    Point3D,
    OrientationEuler3D,
    HiFiCommunicator,
    HiFiConstants
} = from "hifi-spatial-audio";

```